OpTestConfiguration.py

```diff
@@ -584,7 +586,8 @@ def parse_args(self, argv=None):
                                              stdin=subprocess.PIPE,
                                              stderr=sys.stderr,
                                              stdout=sys.stdout,
-                                             shell=True)
+                                             shell=True,
+                                             text=True)
```

Comment:
Just found out this doesn't work on Python 3.6, it's just a cleaner shortcut for existing functionality, so I'll rework it for that